akka-remote/src/main/scala/akka/remote/Remoting.scala

```diff
@@ -409,11 +411,9 @@ private[remote] class EndpointManager(conf: Config, log: LoggingAdapter) extends
   // Mapping between transports and the local addresses they listen to
   var transportMapping: Map[Address, AkkaProtocolTransport] = Map()
 
-  def retryGateEnabled = settings.RetryGateClosedFor > Duration.Zero
-  val pruneInterval: FiniteDuration = if (retryGateEnabled) settings.RetryGateClosedFor * 2 else Duration.Zero
-  val pruneTimerCancellable: Option[Cancellable] = if (retryGateEnabled)
-    Some(context.system.scheduler.schedule(pruneInterval, pruneInterval, self, Prune))
-  else None
+  val pruneInterval: FiniteDuration = settings.RetryGateClosedFor * 2
+  val pruneTimerCancellable: Cancellable =
+    context.system.scheduler.schedule(pruneInterval, pruneInterval, self, Prune)
```

Comment:
ok, I don't know, if it is a bug to allow zero then we can of course change, but otherwise it a bit hard to fail the system startup (and claim that it is a drop in replacement).

This have to go into 2.4.1, unless we have to make another 2.4.0 RC for other reasons, so the situation is pretty much the same there.